RELEASE.md

```diff
@@ -30,6 +30,7 @@
 * Deprecated the "Thanks for supporting contributions" section of release notes to simplify the contribution process; Kedro 0.17.6 is the last release that includes this. This process has been replaced with the [automatic GitHub feature](https://github.com/quantumblacklabs/kedro/graphs/contributors).
 * Fixed a bug where the version on the tracking datasets didn't match the session id and the versions of regular versioned datasets.
 * Fixed an issue where datasets in `load_versions` that are not found in the data catalog would silently pass.
+* Altered the string representation of nodes so that node inputs/outputs order is preserved rather than being alphabetically sorted. 
```

Comment:
```suggestion
* Altered the string representation of nodes so that node inputs/outputs order is preserved rather than being alphabetically sorted.
```